crates/wasm_client_solana/readme.md

```
@@ -31,6 +31,16 @@ When building your crate for the `wasm32-unknown-unknown` target, you need to se
 export RUSTFLAGS='--cfg getrandom_backend="wasm_js"'
 ```
 
+Alternatively you can add this to your `./.cargo/config.toml` file.
+
+```toml
+[target.wasm32-unknown-unknown]
+rustflags = [
+	"--cfg",
+	"get_random_backend=\"wasm_js\"",
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)